examples/3d/transmission.rs

```diff
@@ -0,0 +1,604 @@
+//! This example showcases light transmission
+//!
+//! ## Controls
+//!
+//! | Key Binding        | Action                                    |
+//! |:-------------------|:------------------------------------------|
+//! | `1` / `2`          | Decrease / Increase Diffuse Transmission  |
+//! | `Q` / `W`          | Decrease / Increase Specular Transmission |
+//! | `A` / `S`          | Decrease / Increase Thickness             |
+//! | `Z` / `X`          | Decrease / Increase IOR                   |
+//! | `E` / `R`          | Decrease / Increase Perceptual Roughness  |
+//! | Arrow Keys         | Control Camera                            |
+//! | `O` / `P`          | Decrease / Increase Transmission Steps    |
+//! | `J`/`K`/`L`/`;`    | Transmissive Quality                      |
+//! | `H`                | Toggle HDR                                |
+//! | `D`                | Toggle Depth Prepass (Also disables TAA)  |
+//! | `C`                | Randomize Colors                          |
+
+use std::f32::consts::PI;
+
+use bevy::{
+    core_pipeline::{
+        bloom::BloomSettings, core_3d::TransmissiveQuality, prepass::DepthPrepass,
+        tonemapping::Tonemapping,
+    },
+    pbr::{NotShadowCaster, NotTransmittedShadowReceiver, PointLightShadowMap},
+    prelude::*,
+    render::camera::TemporalJitter,
+    render::view::ColorGrading,
+};
+
+#[cfg(any(not(feature = "webgl2"), not(target_arch = "wasm32")))]
+use bevy::core_pipeline::experimental::taa::{TemporalAntiAliasBundle, TemporalAntiAliasPlugin};
+
+use rand::random;
+
+fn main() {
+    let mut app = App::new();
+
+    app.add_plugins(DefaultPlugins)
+        .insert_resource(PointLightShadowMap { size: 4096 })
+        .insert_resource(AmbientLight {
+            brightness: 0.0,
+            ..default()
+        })
+        .add_systems(Startup, setup)
+        .add_systems(Update, (example_control_system, flicker_system));
+
+    // *Note:* TAA is not _required_ for specular transmission, but
+    // it _greatly enhances_ the look of the resulting blur effects.
+    // Sadly, it's not available under WebGL.
+    #[cfg(any(not(feature = "webgl2"), not(target_arch = "wasm32")))]
```

Comment:
I think it will be available for WebGPU, this is just kinda confusingly expressed.

!webgl2 || !wasm32 == !(webgl2 && wasm32)

Let me change it so that it's easier to read with De Morgan's law.